app/src/org/commcare/activities/CommCareActivity.java

```diff
@@ -980,6 +980,10 @@ protected void showToast(int stringResource) {
         Toast.makeText(this, getLocalizedString(stringResource), Toast.LENGTH_LONG).show();
     }
 
+    protected void showToastMessage(String stringResource) {

```

Comment:
No need for this additional function. It would only be used to show an unlocalized string anyway.